### DISCUSSION TITLE
how to get glider altitude data if matek 411 flight controller has no GPS receiver connected? INAV 5.0 ELRS communication system, I only need stabilization and glider flight altitude data, I don't need GPS

### DISCUSSION THREAD

**Discussion by Andron1967** [iNavFlight/inav#8284] [Q&A]

**Comment by Andron1967** [comment]
I ask the developers to make me a function for determining the height so that the configurator INAV  transmits the barometric sensor values ​​by telemetry without using a GPS receiver

**Reply by OptimusTi** [reply]
TBS is working on the variometer.
